tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -399,7 +399,15 @@ def validateRace():
     """Validate race."""
     return make_validator(
         lambda value: value >= 0 and value <= 2,
-        lambda value: f"{value} is not greater than or equal to 0 or smaller than or equal to 1.",
+        lambda value: f"{value} is not greater than or equal to 0 or smaller than or equal to 2.",
+    )
+
+
+def validateRptMonthYear():
+    """Validate RPT_MONTH_YEAR."""
+    return make_validator(
+        lambda value: value[2:8].isdigit(),
+        lambda value: f"The value: {value[2:8]}, does not follow the YYYYMM format for Reporting Year and Month.",
```

Comment:
@reitermb Any input on this given the recent errors audit? I think we got flagged on saying "the value" in the past.

Reply:
@reitermb 🆙